### PR TITLE
Change Form Submission Type to Text

### PR DESCRIPTION
**Pull request by Vmack**
### What's this PR do?

This pull request is in response to [this slack discussion](https://dosomething.slack.com/archives/C3ASB4204/p1579881161006700) and [pivotal discussion](https://www.pivotaltracker.com/n/projects/2401401/stories/169330883) allowing us to change the form submission type of the `CtaPopoverEmailForm` to `"text"` so that it captures all errors and not just email errors in the interim.

### How should this be reviewed?

👀
should be quick

### Any background context you want to provide?

Adds to PR [1853](https://github.com/DoSomething/phoenix-next/pull/1853)

### Relevant tickets

References [Pivotal #169330883](https://www.pivotaltracker.com/n/projects/2401401/stories/169330883).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: resources/assets/components/pages/GeneralPage/GeneralPage.js [hunk 1]
```diff
@@ -152,6 +152,7 @@ const GeneralPage = props => {
         {slug === 'about/easy-scholarships' ? (
           <DismissableElement
             name="cta_popover_scholarship_email"
+            context={{ contextSource: 'newsletter_scholarships' }}
```
Comment (DFurnes) [inline resources/assets/components/pages/GeneralPage/GeneralPage.js]: I don't see anything in the PR description, so just want to double-check this is an intentional change!

File: resources/assets/components/utilities/CtaPopover/CtaPopoverEmailForm.js [hunk 2]
```diff
@@ -93,7 +93,7 @@ const CtaPopoverEmailForm = ({ handleComplete }) => {
       <form className="email-form form pb-2" onSubmit={handleSubmit}>
         <input
           className="text-field email-form__input"
-          type="email"
+          type="text"
```
Comment (DFurnes) [inline resources/assets/components/utilities/CtaPopover/CtaPopoverEmailForm.js]: Totally down with everyone's decision in the thread/card. 👍 It's worth noting that this means users on mobile devices won't get autocomplete or the email keyboard (with a ".com" button), so may slightly decrease conversion.

File: resources/assets/components/pages/GeneralPage/GeneralPage.js (hunk 1 above)
Comment (DFurnes) [reply resources/assets/components/pages/GeneralPage/GeneralPage.js]: Oh never mind, just saw these were merged in #1853! That's the trick with squashing! 🗜

Comment (Vmack) [reply resources/assets/components/pages/GeneralPage/GeneralPage.js]: yea that's my mistake, I created the new branch after make the change.

File: resources/assets/components/utilities/CtaPopover/CtaPopoverEmailForm.js (hunk 2 above)
Comment (Vmack) [reply resources/assets/components/utilities/CtaPopover/CtaPopoverEmailForm.js]: ohhh wow thanks for pointing that out, you're right @DFurnes

**Pull request merged**